### ISSUE TITLE
Not launching on Linux

### ISSUE THREAD

**Issue by dandevelo** [fman-community/fman#119]
Failed to run on Ubuntu 16.10. Opening the icon does not do anything. 

Checking the System Log revealed this:

```
compiz[2204]: Failed to execute script main
compiz[2204]: Traceback (most recent call last):
compiz[2204]:   File "fman/main.py", line 1, in <module>
compiz[2204]:   File "/home/michael/dev/fman/venv/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 389, in load_module
compiz[2204]:   File "fman/__init__.py", line 4, in <module>
compiz[2204]:   File "/home/michael/dev/fman/venv/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 573, in load_module
compiz[2204]: ImportError: /usr/lib/x86_64-linux-gnu/mesa/libGL.so.1: undefined symbol: drmFreeDevice
```
The only way to get it running was to go into /opt/fman and rename `libdrm.so.2` to `libdrm.so.2.renamed`. This got it to start.

**Comment by mherrmann** [comment]
Thanks! I'll take a look.

**Comment by mherrmann** [comment]
Fixed by not including `libdrm.so.2` any more. It should be available on most systems anyways. The fix will go live with v0.3.8.

**Comment by mherrmann** [comment]
v0.3.8. is live.
